src/routes/todo.routes/createToDo.ts

```diff
@@ -20,7 +20,7 @@ export function createToDoRouter(): Router {
         const { title, description } = req.body;
         todo = createToDo(title, description);
 
-        saveTodo(todo);
+        new todoModel(todo).save().catch((err) => console.log("err", err));
```

Comment:
Saving the todo is business logic, so it should be inside service layer. 
Persisting the todo in the database belongs to data access layer (repository)